account_invoice_line_salesperson/views/account_invoice_views.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="invoice_form" model="ir.ui.view">
+        <field name="name">account.invoice.form</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='invoice_line_ids']//tree//field[@name='name']"
+                position="after"
+            >
+                <field name="salesman_id"/>
```

Comment:
Wasn't sure if should show this in the invoice form.

Reply:
Sorry, it just for confirm work, debug.
I confirmed it works on sql view.